pennylane/_device.py

```diff
@@ -162,6 +166,47 @@ def shots(self, shots):
 
         self._shots = int(shots)
 
+    def define_wire_map(self, wires):
+        """Create the map from user-provided wire labels to the wire labels used by the device.
+
+        The default wire map maps the user wire labels to wire labels that are consecutive integers.
+
+        For example:
```

Comment:
For consistency with other docstrings, should this be moved down to an `**Example**` section below `Returns:`?